test/unit/system/domain_info_test.rb

```diff
@@ -140,7 +140,7 @@ class ApicastInfoTest < ActiveSupport::TestCase
       end
 
       apicast_info = System::DomainInfo.apicast_info('api.3scale.net')
-      assert apicast_info.staging
+      assert apicast_info.staging, "Oracle mystery must be revealed!: #{ProxyConfig.current_versions.by_host('api.3scale.net').take(2).index_by(&:environment)}"
```

Comment:
😄

Reply:
lol the order
![image](https://user-images.githubusercontent.com/11318903/99385049-b03c9100-28d0-11eb-9880-c0f695eae261.png)